### PR TITLE
Fixes monkeys being unable to wear pet collars

### LINKED ISSUE #56952: Can't put pet collars on monkeys any more
## Round ID:
Tested locally

## Testmerges:
N/A

## Reproduction:
Get a collar
Get a monkey
try to pineapple pen that shit
Doesn't work. Thanks, now-officially-human monkeys

Weep as you may never put pun pun's collar on now.

### PR DESCRIPTION
**Pull request by ArcaneDefence**
## About The Pull Request
fixes https://github.com/tgstation/tgstation/issues/56952
Monkeys became a human species, so they were failing the old check

## Why It's Good For The Game
Refactors dropping features is cringe

## Changelog
:cl:
fix: Monkey kind has rediscovered the lost art of wearing a pet collar. Celebrate with Pun Pun!
/:cl:

### CODE REVIEW

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -174,7 +174,7 @@
 	var/tagname = null
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))
+	if(!ismonkey(M))
```
Comment (Fikou) [inline code/modules/clothing/neck/_neck.dm:177]:
```suggestion
	if(ishuman(M) && !ismonkey(M))
```
just to be safe?

**Comment by itseasytosee** [comment]
Turning monkeys into humans has made nothing easier but introduced a shit ton of bugs.

**Comment by optimumtact** [comment]
:-1: do not merge

File: code/modules/clothing/neck/_neck.dm (hunk 1 above)
Comment (ArcaneDefence) [reply code/modules/clothing/neck/_neck.dm:177]: Safe from what, exactly?

**Comment by ArcaneDefence** [comment]
> 
> 
> 👎 do not merge

Then add the label for it

**Comment by optimumtact** [comment]
I can have an opinion if I want, go boil your head

**Comment by Mothblocks** [comment]
image

**Pull request merged**